src/relbot.py

```diff
@@ -55,47 +53,25 @@ def main(
     # Android Components
     if argv[1] == "android-components":
         if argv[2] == "update-main":
-            android_components.update_main(ac_repo, author, dry_run)
+            android_components.update_main(firefox_repo, author, dry_run)
         elif argv[2] == "update-releases":
-            android_components.update_releases(ac_repo, fenix_repo, author, dry_run)
-        elif argv[2] == "create-releases" or argv[2] == "create-release":
-            android_components.create_releases(
-                ac_repo, fenix_repo, author, debug, dry_run
-            )
+            android_components.update_releases(firefox_repo, author, dry_run)
         else:
             print(
-                "usage: relbot android-components <update-{main,releases}|create-releases>"  # noqa E501
+                "usage: relbot android-components <update-{main,releases}>"  # noqa E501
```

Comment:
Nit - can be done in a followup: We should likely rename `android-components` into `firefox-android` because the A-C repo is not used anymore.

Reply:
I pondered doing that, on the other hand we're still updating geckoview/glean/a-s versions used by android-components, so even though the repo name changed this is not actually wrong.